recipes/default.rb

```diff
@@ -43,3 +43,9 @@
 
 node.default['ssh_keys']['alfred'] = 'alfred'
 include_recipe 'ssh-keys'
+
+node.deep_fetch('osl-jenkins', 'gems').each do |g|
+  chef_gem g do # ~FC009
+    compile_time true
+  end
+end
```

Comment:
This should be in the server recipe, not the default recipe.

Reply:
Do you mean the master recipe?